app/src/main/java/com/example/pinkorange/vibrato/LiveWithSettings.java

```diff
@@ -96,6 +112,57 @@ protected void onCreate(Bundle savedInstanceState) {
         }
     }
 
+    private void setMusicControlButton() {
+        playButton.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View view) {
+                if(mAudioPlayer.isPlaying()) {
+                    mAudioPlayer.pause();
```

Comment:
Under the `if` condition can we add a line to change the button's display image to the `round_play_arrow_24` image? Similarly, under the `else` condition can we add a line to change the button's display image to the `round_pause_24` image?

Reply:
I will make those changes as soon as I get home~